docker/nginx.conf

```diff
@@ -1,12 +1,34 @@
+map $sent_http_content_type $expires {
+    # cache everything for 1 year
+    default 1y;
+    # html files shouldn't be cached for single-page applications
+    text/html off;
+}
+
 server {
     listen 80;
     listen [::]:80;
     server_name localhost;
 
+    # serve compressed file if filename.gz exists
+    gzip_static on;
+
+    # Deny IE browsers from going into quirks mode
+    add_header X-UA-Compatible "IE=Edge";
+    # X-Frame-Options is to prevent from clickJacking attack
+    add_header X-Frame-Options SAMEORIGIN;
+    # This header enables the Cross-site scripting (XSS) filter
+    add_header X-XSS-Protection "1; mode=block;";
+    # disable content-type sniffing on some browsers.
+    add_header X-Content-Type-Options nosniff;
```

Comment:
Shouldn't I see these in the response headers? 🧐 

<img width="635" alt="image" src="https://github.com/user-attachments/assets/e8f03590-cfcd-44ea-8fee-1f953ab09e00">

Reply:
Good catch, looks like adding the cache headers kinda broke them. fixed 👍🏻